README.md

```diff
@@ -171,6 +173,37 @@ There is no standard range for touch values aside from requirement to have x axi
 | Offset | Length | Type | Meaning |
 | ------ | ------ | ---- | ------- |
 | 0 | 1 | Unsigned 8-bit | Is touch active (`1` if active, else `0`) |
-| 1 | 1 | Unsigned 8-bit | Touch id (should be the same for one continious touch) |
+| 1 | 1 | Unsigned 8-bit | Touch id (should be the same for one continuous touch) |
 | 2 | 2 | Unsigned 16-bit | Touch X position |
 | 4 | 2 | Unsigned 16-bit | Touch Y position |
+
+## (Unofficial) Information about controller motors
+
+Ask/Reply how many motors in controller for some port.
+
+### Incoming packet structure
+
+| Offset | Length | Type | Meaning |
+| ------ | ------ | ---- | ------- |
+| 0 | 8 | Complex | Controller identification header, same as the incoming packet of "Actual controllers data" |
+
+For every controller motor you should send one packet structured like described below.
+
+### Outgoing packet structure
+
+| Offset | Length | Type | Meaning |
+| ------ | ------ | ---- | ------- |
+| 0  | 11  | Complex | Beginning described above |
+| 11 | 1 | Unsigned 8-bit | Motor id |
```

Comment:
After thinking about it some more - how about just sending motor count per queried controller, with numbering going from zero? This has a pro of better handling situation with packet loss and allows to convey that client supports extension but lacks motors.

Reply:
OK, I have changed. The previous designed with reference to "Information about connected controllers" 😂